### ISSUE TITLE
Implement missing stdlib functions

### ISSUE THREAD

**Issue by JPTIZ** [expertisesolutions/efl#25]
The following must be implemented:

```c
int mkstemp(char* template);
int mkostemp(char* template, int flags);
int mkostemps(char* template, int suffixlen, int flags);
```

**Comment by vtorri** [comment]
why should they be implemented ? mko* functions are not used in the EFL

**Comment by JPTIZ** [comment]
> why should they be implemented ? mko* functions are not used in the EFL

These issues were opened as compile errors emerged (of missing headers or implicit function declaration). Some of them I just checked which functions that header implemented, should've seen this before, since indeed `mkostemps` is not used at all.

For `mkostemp`, I used `git grep -n "mkostemp"` and it returned:

`src/modules/ecore_buffer/shm/ecore_buffer_shm.c:73:   fd = mkostemp(name, O_CLOEXEC);`

Should that module be skipped in Windows?
